src/ui/public/vis/agg_config_result.js

```diff
@@ -3,7 +3,7 @@ define(function () {
 
   function AggConfigResult(aggConfig, parent, value, key) {
     this.key = key;
-    this.value = value;
+    this.value = value === null ? false : value; // restrict to primitive for sorting in Angular
```

Comment:
I think that alternating the value represented by the `AggConfigResult` is a little heavy handed, especially since I'm pretty sure that `null` is a perfectly valid value for `AggConfigResult` objects to represent. I think it would be more appropriate to apply a fix to the [`valueGetter()`](https://github.com/spalger/kibana/blob/6f1cd39f65fe6aa217413ea4c35d55675ac017fe/src/ui/public/paginated_table/paginated_table.js#L54-L59) that is providing values to the sorting algorithm.